guides/common/modules/proc_enabling-pull-based-transport.adoc

```diff
@@ -0,0 +1,66 @@
+[id="enabling-pull-based-tranport_{context}"]
+= Enabling Pull-based Transport
+
+If your {Project} or {SmartProxy} is preinstalled, use the following procedure to enable pull-based transport.
+
+.Procedure
+
+. To install the remote execution script, run:
+ifdef::satellite[]
+----
+{satellite-installer} --foreman-proxy-plugin-remote-execution-script-mode=pull-mqtt
+----
+endif::[]
+ifdef::foreman,orcharhino,katello[]
+----
+{foreman-installer} --foreman-proxy-plugin-remote-execution-script-mode=pull-mqtt
+----
+endif::[]
+
+// Start this list from 2.
+[start=2]
```

Comment:
Why is there [start=2] ? We don't use this. The order list should work without this. You just need to add + character after: ... run:
+
ifdef